### PR TITLE
With nodemon

### PR DESCRIPTION
**Pull request by vinaypuppal**
This is the PR for using nodemon with development server as discussed in issue #6 

Changes made
- Added nodemon as npm dev dependency
- Moved server.js to server folder
- Fixed relative paths to serve index.html & public folder
- Fixed main entry point in package.json
- Added `npm run start-dev` script with nodemon to watch server folder
- update README for changed npm scripts i.e how to start dev server with nodemon

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -1,14 +1,16 @@
 var express = require('express');
```
Comment (JordyVlassembrouck) [inline server/server.js]: I wouldn't use enters between variables, especially not when they belong together (both requires).

File: server/server.js [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
+var path = require('path');
+
```
Comment (JordyVlassembrouck) [inline server/server.js]: All these declarations can actually be underneath each other.

**Comment by JordyVlassembrouck** [comment]
@vinaypuppal two comments.

**Comment by vinaypuppal** [comment]
@JordyVlassembrouck changed as you told. Further any changes required??

**Comment by rhgksrua** [comment]
Looks good.  Once @JordyVlassembrouck takes a look, you can merge it.

**Comment by vinaypuppal** [comment]
@JordyVlassembrouck @rhgksrua Any changes required or can i merge ??

**Comment by rhgksrua** [comment]
@JordyVlassembrouck is no longer on this project, I think.  You can merge it.  Looks like no one has anything more to say about this matter.

**Pull request merged**